### ISSUE TITLE
Is there a way to group the rows?

### ISSUE THREAD

**Issue by opensourcegeek** [AllenFang/react-bootstrap-table#751]
If I have multiple rows with "A" as value for one of the columns, it would be nice to be able to hide/show or expand/collapse rows that has column "A" in them. Also it would mean you would have to use this value to sort I think. Have anyone tried doing in any forks of this project?

**Comment by AllenFang** [comment]
@opensourcegeek, sorry we don't have this kind of feature and I cant support this feature in near future cause of I'm focus on ```v3.0.0``` development, so PR is welcome. Thanks you.  I'll close this until someone that interesting to contribute this feature.

**Comment by AllenFang** [comment]
@opensourcegeek, in ```v2.6.0```, we support expand row, check [example](https://github.com/AllenFang/react-bootstrap-table/blob/master/examples/js/expandRow/expandRow.js). We will update document in these day

**Comment by opensourcegeek** [comment]
@AllenFang it's cool, I'll take a look now - there's a typo in your example

```
16:55 $ npm run start

> react-bootstrap-table@2.6.0 start /home/opensourcegeek/projects/github/react-bootstrap-table-orig/react-bootstrap-table
> gulp example-server

[16:55:22] Using gulpfile ~/projects/github/react-bootstrap-table-orig/react-bootstrap-table/gulpfile.js
[16:55:22] Starting 'example-server'...
[16:55:22] Finished 'example-server' after 54 ms
Listening at localhost:3004
Time: 41338ms
        Asset     Size  Chunks       Chunk Names
app.bundle.js  8.92 MB       0       app
   vendors.js   670 kB       1       vendors
chunk    {0} app.bundle.js (app) 3.4 MB {1} [rendered]
chunk    {1} vendors.js (vendors) 225 kB [rendered]

ERROR in ./examples/js/expandRow/demo.js
Module not found: Error: Cannot resolve 'file' or 'directory' ./ExpandRow in /home/opensourcegeek/projects/github/react-bootstrap-table-orig/react-bootstrap-table/examples/js/expandRow
 @ ./examples/js/expandRow/demo.js 13:17-39
webpack: bundle is now VALID.

```
The import in ./examples/js/expandRow/demo.js,  
`import ExpandRow from './ExpandRow';`

There is no ExpandRow in that directory so I changed it to expandRow as below and it builds fine.
`import ExpandRow from './expandRow';`

**Comment by opensourcegeek** [comment]
@AllenFang It looks good - how are you planning to make `selection` work with this table? I will play with it and see how it goes, I want this "expanded table" to be selectable as well. Is it possible?

**Comment by AllenFang** [comment]
> I want this "expanded table" to be selectable as well. Is it possible?

sure, you can try it. Feel free to give any feedback to me 👍

**Comment by opensourcegeek** [comment]
I took a closer look this morning, it looks good to me - I'd probably have to wire up selection with my app differently so that I can select all the rows in sub table (expanded row) when user selects a row in primary table it is do-able. 

I've forked this repo and added some changes to how search works as well (added exact match search) so I might have to merge and see if everything holds together fine for this feature! Thanks for letting me know.
